VERSION

```diff
@@ -1 +1 @@
-v0.15
+v0.16
```

Comment:
Why do we need to up? Because of `type Admitter interface` change?

Reply:
Yeah, I thought we should bump minor version due to dependency change

Reply:
Do you think it's relevant for library users?
Normally I would increase minor version to notify library users.

Reply:
Technically it breaks API (changes Admitter interface).
I do not think anyone uses this package, maybe we should also unexport its symbols

Reply:
> maybe we should also unexport its symbols

I've made symbols unexported